src/Nethermind/Nethermind.Core/ExecutionRequest/ExecutionRequest.cs

```diff
@@ -0,0 +1,141 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using Nethermind.Core.Collections;
+using Nethermind.Core.Crypto;
+using Nethermind.Core.Extensions;
+
+namespace Nethermind.Core.ExecutionRequest;
+
+public enum ExecutionRequestType : byte
+{
+    Deposit = 0,
+    WithdrawalRequest = 1,
+    ConsolidationRequest = 2
+}
+
+public class ExecutionRequest
+{
+    public byte RequestType { get; set; }
+    public byte[]? RequestData { get; set; }
+
+    public override string ToString() => ToString(string.Empty);
+
+    public string ToString(string indentation) => @$"{indentation}{nameof(ExecutionRequest)}
+            {{{nameof(RequestType)}: {RequestType},
+            {nameof(RequestData)}: {RequestData!.ToHexString()}}}";
+}
+
+public static class ExecutionRequestExtensions
+{
+    public const int depositRequestsBytesSize = 48 + 32 + 8 + 96 + 8;
+    public const int withdrawalRequestsBytesSize = 20 + 48 + 8;
+    public const int consolidationRequestsBytesSize = 20 + 48 + 48;
```

Comment:
would be good to use other constants to calculate this one, like `Address.Size` ect